OsmAnd/src/net/osmand/plus/configmap/ConfigureMapMenu.java

```diff
@@ -262,7 +253,7 @@ private void createRouteAttributeItems(@NonNull List<RenderingRuleProperty> cust
 				adapter.addItem(createCycleRoutesItem(activity, attrName, property, nightMode));
 			} else if (HIKING_ROUTES_OSMC_ATTR.equals(attrName)) {
 				adapter.addItem(createHikingRoutesItem(activity, attrName, property, nightMode));
-			} else if (SHOW_MTB_ROUTES.equals(attrName)) {
+			} else if (SHOW_MTB_ROUTES_ATTR.equals(attrName)) {
```

Comment:
From
SHOW_MTB_ROUTES_ATTR.equals(attrName)
TO
MTB.getRenderingProperty().equals(attrName);